servers/rendering/rasterizer_rd/shaders/particles.glsl

```diff
@@ -314,6 +314,11 @@ void main() {
 
 		if (restart) {
 			PARTICLE.is_active = FRAME.emitting;
+			PARTICLE.xform = mat4(
+					vec4(1.0, 0.0, 0.0, 0.0),
+					vec4(0.0, 1.0, 0.0, 0.0),
+					vec4(0.0, 0.0, 1.0, 0.0),
+					vec4(0.0, 0.0, 0.0, 1.0));
```

Comment:
I think you can just use `mat4(1.0)` for an identity matrix: https://www.khronos.org/opengl/wiki/Data_Type_(GLSL)#Matrix_constructors

> If a matrix is constructed with a single scalar value, then that value is used to initialize all the values along the diagonal of the matrix; the rest are given zeros. Therefore, mat4(1.0) is a 4x4 identity matrix.

Reply:
Thanks, fixed.